Twitter Support Reimplementation  in Swift

lennet opened the pull request:
This is a reimplementation of the Twitter integration with using the critical maps api instead of a direct connection to twitter. I also made deleted some unused icons and made some changes to the pod file so that the project is finally back to 0 warnings 🎉

<img width="545" alt="bildschirmfoto 2019-01-28 um 22 53 12" src="https://user-images.githubusercontent.com/7677738/51869155-0503c600-2350-11e9-9590-1cc31a813bd8.png">

We should maybe consider also showing the date of a tweet next to the hour and minute.

fixes #32

zutrinken commented:
Here is an updated screen (please ignore NavBar and missing TabBar) with relative dates and username. It's similar to the official iOS twitter app. Optional highlighting hashtags (not sure if it makes any sense in our case).

![6735664265243](https://user-images.githubusercontent.com/888679/51980588-c6295980-2490-11e9-8cd3-4d24c9e72211.png)

lennet commented on CriticalMass/TwitterManager.swift:
```diff
@@ -0,0 +1,63 @@
+
+    var updateTweetsCallback: (([Tweet]) -> Void)?
+
+    init(networkLayer: NetworkLayer, url: URL = URL(string: "https://api.criticalmaps.net/twitter")!) {
```
Good point! I added a Constants file

lennet commented:
> Here is an updated screen (please ignore NavBar and missing TabBar) with relative dates and username. It's similar to the official iOS twitter app. Optional highlighting hashtags (not sure if it makes any sense in our case).
> 
> ![6735664265243](https://user-images.githubusercontent.com/888679/51980588-c6295980-2490-11e9-8cd3-4d24c9e72211.png)

![2019-01-30 20-50-38 2019-01-30 20_51_52](https://user-images.githubusercontent.com/7677738/52008634-4d96bd00-24d1-11e9-9f70-64da3d9c7b5a.gif)
 I also added highlighting for hashtags and usernames but they aren't interactive though for now. We could link them to the Twitter app/ Website in the future

zutrinken commented:
> We could link them to the Twitter app/ Website in the future

That's a good idea. But I guess it's better not highlighting them unless they're interactive. Else it's misleading.

One more thing: Regular URLs. They should be linked as well then.

lennet commented:
> > We could link them to the Twitter app/ Website in the future
> 
> That's a good idea. But I guess it's better not highlighting them unless they're interactive. Else it's misleading.
> 
> One more thing: Regular URLs. They should be linked as well then.

makes sense! I removed the highlighting and added an Issue for that feature
